### ISSUE TITLE
File endings in sdist uploaded on pypi are not consistent

### ISSUE THREAD

**Issue by nehaljwani** [pyinstaller/pyinstaller#5759]
```
$ wget 'https://pypi.io/packages/source/p/pyinstaller/pyinstaller-4.2.tar.gz'
$ tar -xf pyinstaller-4.2.tar.gz pyinstaller-4.2/bootloader/wscript && file $_
pyinstaller-4.2/bootloader/wscript: Python script, ASCII text executable
```

```
$ wget 'https://pypi.io/packages/source/p/pyinstaller/pyinstaller-4.3.tar.gz'
$ tar -xf pyinstaller-4.3.tar.gz pyinstaller-4.3/bootloader/wscript && file $_
pyinstaller-4.3/bootloader/wscript: Python script, ASCII text executable, with CRLF line terminators
```

**Comment by bwoodsend** [comment]
Why on earth does that matter? Python is more than capable of running a script with the wrong line endings for the OS it is on.

**Comment by nehaljwani** [comment]
It matters because it breaks build pipelines for distributions that apply patches over original source code, and the breakage did happen at conda-forge. For now, I've updated the patch we use to make it DOS-like and proceeded ahead, to unblock the release.

I simply wanted to report this change. Feel free to close.

**Comment by Legorooj** [comment]
`git config core.autocrlf` really messes with my dev environment lol... I'll just rebuild the sdists from WSL.

**Comment by bwoodsend** [comment]
Would it help if we re-pushed the sdist with `\n` line endings or is the damage already done?

**Comment by nehaljwani** [comment]
Please don't re-push! Perhaps streamlining the sdist release on PyPI with [a CI](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/) would help solve the issue long term.

**Comment by bwoodsend** [comment]
Historically, we've made a point of never uploading anything we haven't built ourselves on our own machines because we can't verify that CI machines haven't been infected with something malicious. I doubt we'll maintain that policy for much longer though - we've already switched to CI/CD for the hooks repo.

**Comment by Legorooj** [comment]
Huh I built these in WSL. Damn, thought that would fix the issue. I'll automate the release pipeline in the future so the wheels are built on a Linux machine @nehaljwani.

**Comment by nehaljwani** [comment]
Looks like this has been fixed in the 4.5.1 release. Closing!
